app/assets/javascripts/index/contextmenu.js

```diff
@@ -177,7 +177,7 @@ class ContextMenu {
   }
 
   _render(items) {
-    const $menuList = $("<ul>").addClass("dropdown-menu show shadow cm_dropdown_menu");
+    const $menuList = $("<ul>").addClass("dropdown-menu list-group show shadow cm_dropdown_menu");
```

Comment:
I don't think it's a good idea to try to make this styled as *both* a dropdown menu and also as a list-group. It shouldn't be both, these are two different components.

```diff
-    const $menuList = $("<ul>").addClass("dropdown-menu list-group show shadow cm_dropdown_menu");
+    const $menuList = $("<ul>").addClass("dropdown-menu show shadow cm_dropdown_menu");
```